REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,96 @@
+
+You can find the list of all ```Infra``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#infra-scope).
+
+### ```Special```
+
+Special repositories are non-archived repositories that do not fit neatly into ```Core```, ```Ecosystem```, or ```Infra```. They are generally not designed to be used by adoptors.
+
+They include the [Community](https://github.com/cilium/community) and [.github](https://github.com/cilium/.github) repository, template repositories like [design-cfps](https://github.com/cilium/design-cfps), as well as all forks.
+
+You can find the list of all ```Special``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#special-scope).
+
+### ```Historical```
+
+Historical repositories are considered deprecated, are no longer maintained, and are converted to an archived, read-only state. They are not recommended for use by adoptors.
+
+Repositories that have been inactive for quite some time may be good candidates for archival. They can be proposed through the same process as other [changes of scope](#change-of-scope).
+
+You can find the list of all ```Historical``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#historical-scope).
+
+## Lifecycle
+
+### Voting
+
+All changes that occur within the lifecycle of a repository have the same process:
+
+- An issue is opened via the appropriate issue template ([New Repository Template]() or [Change of Scope Template]())
+- Once the issue is opened, ample opportunity for discussion among both committers and the larger Cilium community is provided (for example, 1 week)
+- If there are no objections among committers during this time and any relevant criteria are met (for example, the criteria found in [Addition](#addition)), the change is approved
+- If there are any objections from committers, the committers decide if the change should be approved via a simple majority vote
+
+### Addition
+
+New repositories of any type can be proposed by using the [New Repository Template]() and can be approved via the process as described in [Voting](#voting).
+
+In order for a new repository to be approved, it must also meet the following criteria:
+
+- Approved by at least two committers, with the assumption that any committer who approves is taking on the responsibility of maintaining the new repository
+- Basic documentation provided on the README
+- Adheres to one of the [approved license policies](https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md)
+
+Any new repository must be added into the appropriate list in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md).
```

Comment:
We may want to just turn these steps into part of the checklist on the issue template, for instance under a separate section "To be filled by a maintainer". Then the set of steps to configure the repository correctly would be in-line with the request, which would make it easier to ensure that the policy is followed.

Reply:
Added this step to both forms